wordpress-mobile/AztecEditor-Android
# Implement indent and outdent across multiple item types

### Fix
In this PR I've added support for indent and outdent for headings, preformat, quotes and paragraphs (as well as text without any spans). The logic adds a `\t` element at the beginning of each selected line on indent and removes the `\t` if present on outdent. 


### Test
1. Make sure you add the `indent` and `outdent` toolbar actions in `MainActivity`
2. Run the app
3. Try indenting/outdenting everything including selection over multiple items 
4. Notice the selection is moving correctly when indenting/outdenting
5. Notice the indent/outdent buttons are enabled when they should be 

### Review
@AmandaRiu 

Make sure strings will be translated:
